docs/adr-0003-discrete_metrics_table_structure.md

```diff
@@ -20,7 +21,9 @@ We want Jetstream to support discrete metric execution ([proposal doc](https://g
 
 ## Decision Outcome
 
-**Option 2** is the chosen option. We can preserve the schemas of the views by pivoting the results tables, and we save a lot on complexity during normal execution as compared to Option 1. Option 3 is very similar but creates additional clutter in the form of many tables.
+**Option 3** is the chosen option.
+
+We can preserve the data type of the metric values in BigQuery, and the schema broadly stays the same (client info + column for metric values -- though split across many tables). Option 2 is very similar, but pivots the data so that it can be stored row-wise in a single table (metric_slug: metric_value), and importantly requires many additional DML operations to delete existing metric results because it cannot simply truncate the entire table on each write like we do now, and can do in Option 3.
```

Comment:
Is it worth adding somewhere here that we are going with a hybrid approach of option 3 and the way it has been running. Discrete metrics are only used when explicitly executed with that option, otherwise things are run as is.

Reply:
Yea good point, I'll update it.